ci/build-docker.sh

```diff
@@ -105,6 +91,21 @@ if [ "${GENERATE_DOCS}" = "yes" ]; then
   make doxygen-docs
 fi
 
+# Some of the sanitizers only emit errors and do not change the error code
```

Comment:
What's the rationale for moving this after the doc generation? Just curious.

Reply:
I wanted to group all the code that scans logs for errors until the end.